README.md

```diff
@@ -53,48 +48,14 @@ Browse the [HASH roadmap] for more information about upcoming features and relea
 
 ## [![a](/.github/assets/gh_icon_repo-structure_20px-base.svg)][gh-repo-structure] &nbsp; About this repository
 
-### Top-level layout
-
 This repository's contents is divided across several primary sections:
 
-- [**`/apps`**](/apps) contains the primary code powering our runnable [applications](#applications)
-- [**`/blocks`**](/blocks) contains our public _Block Protocol_ [blocks](#blocks)
-- [**`/infra`**](/infra) houses deployment scripts, utilities and other [infrastructure](#infrastructure) useful in running our apps
-- [**`/libs`**](/libs) contains [libraries](#libraries) including npm packages and Rust crates
+- [**`/apps`**](/apps) contains the primary code powering our runnable [applications](https://github.com/hashintel/hash/tree/main/apps#applications)
```

Comment:
Out of interest, why change the links?

Reply:
Fewer places to maintain the lists (sources of truth).